recipes/emsdk/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,8 @@
+#include <cstdlib>
+#include <iostream>
+
+int main()
+{
+    std::cout << "Bincrafters\n";
+    return EXIT_SUCCESS;
```

Comment:
Is this file used?

Reply:
This is used to test the package. See conanfile in testpackage.